org.featherwallet.Feather.yaml

```diff
@@ -0,0 +1,129 @@
+app-id: org.featherwallet.Feather
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: feather
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=~/Monero:create
+modules:
+  - name: unbound
+    buildsystem: autotools
+    config-opts:
+      - '--without-pyunbound'
+      - '--without-pythonmodule'
+      - '--with-libevent=no'
+      - '--with-pthreads'
+      - '--with-libunbound-only'
+    sources:
+      - type: git
+        url: 'https://github.com/NLnetLabs/unbound.git'
+        tag: release-1.17.1
+        commit: 90831af981221bbce1cd7b15055562336760e484
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh
+      - ./b2
+            --with-atomic
+            --with-system
+            --with-filesystem
+            --with-thread
+            --with-date_time
+            --with-chrono
+            --with-regex
+            --with-serialization
+            --with-program_options
+            --with-locale
+            --prefix=${FLATPAK_DEST}
+            variant=release link=shared debug-symbols=off runtime-link=shared
+            install
+    sources:
+      - type: archive
+        url: 'https://downloads.sourceforge.net/project/boost/boost/1.78.0/boost_1_78_0.tar.bz2'
+        sha256: 8681f175d4bdb26c52222665793eef08490d7758529330f98d3b29dd0735bccc
+
+  - name: zbar
+    buildsystem: autotools
+    config-opts:
+      - '--without-imagemagick'
+      - '--without-gtk'
+      - '--without-python'
+      - '--without-dbus'
+    sources:
+      - type: archive
+        url: 'https://www.linuxtv.org/downloads/zbar/zbar-0.23.90.tar.gz'
+        sha256: ff857dd7e3dbe043dac3765b5182c91dfd0477800713a75d15287d797cee60fa
+
+  - name: qrencode
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: 'https://github.com/fukuchi/libqrencode.git'
+        tag: v4.1.1
+        commit: 715e29fd4cd71b6e452ae0f4e36d917b43122ce8
+
+  - name: zip
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: 'https://github.com/nih-at/libzip.git'
+        tag: v1.9.2
+        commit: 5532f9baa0c44cc5435ad135686a4ea009075b9a
+
+  - name: sodium
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: 'https://github.com/jedisct1/libsodium.git'
+        tag: '1.0.18-RELEASE'
+        commit: 940ef42797baa0278df6b7fd9e67c7590f87744b
+
+  - name: feather
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -D BUILD_TAG="linux-x64"
+      - -D TOR_DIR=Off
+      - -D TOR_VERSION=Off
+      - -D CHECK_UPDATES=Off
+      - -D WITH_SCANNER=Off
+      - -D REPRODUCIBLE=Off
+    post_install:
+      - 'mv -v /app/feather /app/bin/feather'
+    sources:
+      - type: git
+        url: 'https://github.com/feather-wallet/feather'
+        tag: '2.4.4'
+        commit: ffd6d332daa2f50da3f434de35837204d973ca39
+      - type: shell
+        commands:
```

Comment:
this should all be in post-install, not a shell source.

Reply:
just that with `builddir` set to true files in post-install need to be referenced with `../` to be relative to the top source.